src/rosdistro/freeze_source.py

```diff
@@ -100,7 +118,7 @@ def _worker(work_queue):
             work_queue.task_done()
 
         except subprocess.CalledProcessError:
-            print("Non-zero return code for: %s" % ' '.join(cmd), file=sys.stderr)
+            print("No information could be retrieved for repo %s" % source_repo.url)
```

Comment:
Right now we just spew this error to the console but otherwise behave as if successful. A further improvement at some point could track these failures and perhaps abort the overall operation, returning an error code.